wsmatrix@martin.zurowietz.de/overview/workspacesView.js

```diff
@@ -0,0 +1,137 @@
+const Self = imports.misc.extensionUtils.getCurrentExtension();
+const {Clutter} = imports.gi;
+
+const GWorkspacesView = imports.ui.workspacesView;
+const Util = Self.imports.util;
+const Main = imports.ui.main;
+
+var FitMode = {
```

Comment:
Should this be a `let` or `const`? I think `var` is deprecated these days.

Reply:
It's not really deprecated, it just has different use cases (and `let` and `const` make sense more often). In this case the code is just taken from the [GNOME Shell source](https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/e6c903f4aa8d3ded3a3bfd8ff1167e5142340d48/js/ui/workspacesView.js#L93). But thanks for having a look :wink: